app/workers/detect_invariants.rb

```diff
@@ -17,7 +18,7 @@ def detect_application_choices_in_old_states
         One or more application choices are still in `awaiting_references` or
         `application_complete` state, but all these states have been removed:
 
-        #{choices_in_wrong_state.map(&:id).join("\n")}
+        #{choices_in_wrong_state.map(&:id).sort.join("\n")}
```

Comment:
Is there a reason why we sort this output but not the other one?

Reply:
Just that the associated spec expects ordered output. No reason not to sort the others tho, probably nicer that way. Will add. :+1: